pdns/dnsdistdist/dnsdist-settings-definitions.yml

```diff
@@ -2090,7 +2090,7 @@ load_balancing_policies:
     - name: "hash_perturbation"
       type: "u32"
       default: "0"
-      lua-name: "setWHashedPertubation"
+      lua-name: "setWHashedPerturbation"
```

Comment:
I'm not quite sure how this relates, but I'm hoping this change is ok.

Reply:
We will need to re-generate the YAML settings so this change is taken into account. It should have been done automatically if you have YAML configuration support enabled. I can apply the changes for you if needed, just let me know.